### PR TITLE
doc: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RajaMBZ**
-Addition of (much-needed) commas.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Communication Channels
 ----------------------
 
 Most communication about Bitcoin Core development happens on IRC, in the
-#bitcoin-core-dev channel on Freenode. The easiest way to participate on IRC is
+#bitcoin-core-dev channel on Freenode. The easiest way to participate in IRC is
```
Comment (sipa) [inline CONTRIBUTING.md]: This is incorrect.

Comment (RajaMBZ) [reply CONTRIBUTING.md]: Reverted back that particular change in the 2nd commit.

**Comment by fanquake** [comment]
Please [squash your commits](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits).

**Comment by sipsorcery** [comment]
NACK use of commas is very subjective (particularly before ["and"](https://www.grammarly.com/blog/comma-before-and/) on lists) and IMHO these additions don't improve readability.

@RajaMBZ effort appreciated though and not meant to discourage. Just adding my 2c.

**Comment by michaelfolkson** [comment]
NACK also. I don't agree with the use of at least one of these commas.

More importantly I don't think this is a priority for some of the participants in this thread.

@RajaMBZ: As a new contributor I'm happy to discuss ideas with you on how to contribute to the docs if you're interested in doing so going forward. For now I recommend closing this particular PR.

**Comment by RajaMBZ** [comment]
Well, thank you all for providing your valuable feedback & suggestion(s) in this regard. I'm now closing this PR.

**Pull request closed without merging**